templates/web-application/.stylelintignore

```diff
@@ -1 +1,11 @@
-*
+# Stylelint ignore don't support ! (not) expressions
```

Comment:
Could you add an issue with your findings about this in https://github.com/workleap/wl-web-configs please?

So we can adjust the documentation accordingly.

Reply:
New issue created: https://github.com/workleap/wl-web-configs/issues/39